firmware/thruna/PDM/src/app/app_powerManager.c

```diff
@@ -1,49 +1,131 @@
 #include "app_powerManager.h"
 #include "app_canRx.h"
+#include "app_retryHandler.h"
+#include "app_canTx.h"
+#include <stdio.h>
 
 typedef struct
 {
-    bool efuses[NUM_EFUSE_CHANNELS];
+    RetryConfig retry_configs[NUM_EFUSE_CHANNELS];
 } PowerStateConfig;
 
+static RetryData retry_data[NUM_EFUSE_CHANNELS];
+
 static const PowerStateConfig power_states_config[NUM_POWER_STATES] = {
     [POWER_MANAGER_SHUTDOWN] = {
-        .efuses = {
-            [EFUSE_CHANNEL_AIR] = true,
-            [EFUSE_CHANNEL_LVPWR] = true,
-            [EFUSE_CHANNEL_EMETER] = false,
-            [EFUSE_CHANNEL_AUX] = false,
-            [EFUSE_CHANNEL_DRS] = false,
-            [EFUSE_CHANNEL_FAN] = false,
-            [EFUSE_CHANNEL_DI_LHS] = false,
-            [EFUSE_CHANNEL_DI_RHS] = false,
+        .retry_configs = {
+            [EFUSE_CHANNEL_AIR] = {true, RETRY_PROTOCOL_AIR, 1},
+            [EFUSE_CHANNEL_LVPWR] = {true, RETRY_PROTOCOL_LV, 1},
+            [EFUSE_CHANNEL_EMETER] = {false, RETRY_PROTOCOL_EMETER, 3},
+            [EFUSE_CHANNEL_AUX] = {false, RETRY_PROTOCOL_AUX, 3},
+            [EFUSE_CHANNEL_DRS] = {false, RETRY_PROTOCOL_DRS, 3},
+            [EFUSE_CHANNEL_FAN] = {false, RETRY_PROTOCOL_FANS, 3},
+            [EFUSE_CHANNEL_DI_LHS] = {false, RETRY_PROTOCOL_DI_LHS, 1},
+            [EFUSE_CHANNEL_DI_RHS] = {false, RETRY_PROTOCOL_DI_RHS, 1},
```

Comment:
Consider using designated initializers (label each variable when you initialize a struct) since it's not clear what each parameter means unless you manually open `app_retryHandler.h`. 

For example, this snippet for the efuses is a lot more readable:
```c
static const EfuseConfig efuse_configs[NUM_EFUSE_CHANNELS] = {
    [EFUSE_CHANNEL_AIR] = {
        .enable_gpio = {
            .port = EN_AIR_GPIO_Port,
            .pin = EN_AIR_Pin
        },
        .stby_reset_gpio = {
            .port = FR_STBY_AIR_LVPWR_GPIO_Port,
            .pin = FR_STBY_AIR_LVPWR_Pin,
        },
        .cur_sns_adc_channel = ADC_1_CHANNEL_9,
    },
```